sdk/storage/azdatalake/common.go

```diff
@@ -0,0 +1,31 @@
+//go:build go1.18
+// +build go1.18
+
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+package azdatalake
+
+import (
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore"
+	"github.com/Azure/azure-sdk-for-go/sdk/storage/azdatalake/internal/generated"
+)
+
+// ClientOptions contains the optional parameters when creating a Client.
+type ClientOptions struct {
```

Comment:
Can this be moved to `internal/base` package. Just to be uniform with azblob and azfile.